dashboard/src/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>The Dread Pirate Robot's Dashboard++ Sorta</title>
+  <link rel="stylesheet" href="lib/bootstrap-v4.0.0-alpha.6.min.css">
+  <link rel="stylesheet" href="main.css">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
```

Comment:
When we're connected to the boat wifi, we don't have internet access. Let's stick with the default `sans-serif` font, even if it looks ugly.